test/helper.test.js

```diff
@@ -161,3 +161,10 @@ test('should ensure can access all decorators', async t => {
   t.teardown(() => app2.close())
   t.ok(app2.test)
 })
+
+test('should return the fastify instance when using serverModule', async t => {
+  const argv = ['']
+  const app = await helper.build(argv, {}, {}, require('../examples/plugin.js'))
+  t.teardown(() => app.close())
+  t.notOk(app.server.listening)
```

Comment:
The test could be clearer

```suggestion
  const app = await helper.build(argv, { skipOverride: true }, {}, (app, opts, next) => {
    app.decorate('foo', 'bar')
    next()
  })
  t.teardown(() => app.close())
  t.equals(app.foo, 'bar')
```